met-web/src/components/engagement/new/view/index.tsx

```diff
@@ -3,15 +3,19 @@ import { EngagementHero } from './EngagementHero';
 import { EngagementDescription } from './EngagementDescription';
 import { EngagementContentTabs } from './EngagementContentTabs';
 import { EngagementSurveyBlock } from './EngagementSurveyBlock';
+import { SuggestedEngagements } from './SuggestedEngagements';
 
 export const ViewEngagement = () => {
     return (
-        <main>
-            <EngagementHero />
-            <EngagementDescription />
-            <EngagementContentTabs />
-            <EngagementSurveyBlock />
-        </main>
+        <>
+            <main>
+                <EngagementHero />
+                <EngagementDescription />
+                <EngagementContentTabs />
+                <EngagementSurveyBlock />
+            </main>
+            <SuggestedEngagements />
```

Comment:
Is there a reason this is outside the `main` tag? I think generally that tag should encompass all site content, including sections